CHANGELOG.md

```diff
@@ -1,9 +1,35 @@
 # Changelog
 
+## 9.0.0 (2024-06-10)
+
+### Enhancements
+
+- Added the `Bugsnag-Integrity` header to outgoing Bugsnag requests. [#797](https://github.com/bugsnag/bugsnag-unity/pull/797)
```

Comment:
```suggestion
- Added the `Bugsnag-Integrity` header to outgoing Bugsnag requests to avoid extraneous releases being generated if payloads are altered. [#797](https://github.com/bugsnag/bugsnag-unity/pull/797)
```